src/server/StudioManagementSystem/Controllers/OwnerContactsController.cs

```diff
@@ -21,7 +21,7 @@ public OwnerContactsController(IOwnerContactRepository ownerContactRepository, I
     public record OwnerContactRequest(List<Guid> Ids);
 
     [HttpPost]
-    public ActionResult<List<OwnerDto>> GetOwners(OwnerContactRequest request)
+    public List<OwnerDto?> GetOwners(OwnerContactRequest request)
```

Comment:
We would return a list of owners or an empty list here. Propagating null isn't fun for any consumer here, as it tells them nothing useful

Reply:
^ why this is now null is another question entirely. I cannot see any change to the owner repo'. Is this is a bug? It doesn't look intentional.